### PR TITLE
Fix database lock not being deleted on errors; better document UUID usage.

### PR DESCRIPTION
**Pull request by dnarc**
* Expand documentation on how to find correct UUID values for transfer source and AIP storage destinations to allow for actual full automation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ This script can be modified, to adjust how automate transfers works.  The full s
 * `--files`: If set, start transfers from files as well as folders.
 * `--hide`: If set, hides the Transfer and SIP once completed.
 
+Configuration - Getting Correct UUIDs
+-------------------------------------
```
Comment (Hwesta) [inline README.md]: Please change this to a 3rd level header.  Eg `### Getting Correct UUIDs`

File: README.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ This script can be modified, to adjust how automate transfers works.  The full s
...
+
+The automation script relies on installation-specific UUIDs. To obtain the transfer source UUID for script invocation, visit the 'Transfer Source' tab in the Archivematica Storage Space web dashboard. If a row is marked as a transfer souce its UUID value will be valid for a transfer. To fully automate AIP storage transfer, the processingMCP.xml file used in your transfer must be changed. For example, if you are using the defaultProcessingMCP.xml file provided by this tool in the pre-transfer directory, you need to modify the following section:
+
+    <preconfiguredChoice>
+      <appliesTo>b320ce81-9982-408a-9502-097d0daa48fa</appliesTo>
+      <goToChain>/api/v2/location/99d0466e-173d-4b67-98a3-490198e56925/</goToChain>
+    </preconfiguredChoice>
+
+The UUID after /location/ should correspond to a UUID of an AIP Storage location, also findable in the 'Transfer Source' tab in the Archivematica Storage Space web dashboard. Non-Archivematic Storage Space AIP storage destinations will likely need more configuration.
```
Comment (Hwesta) [inline README.md]: Archivematica's normal workflow doesn't support storing AIPs outside the Storage Service. Can you change this to `Non-Archivematic Storage Space AIP storage destinations are unsupported.` or remove it?

File: README.md [hunk 3]
```diff
@@ -42,6 +42,18 @@ This script can be modified, to adjust how automate transfers works.  The full s
...
 
+Configuration - Getting Correct UUIDs
+-------------------------------------
+
+The automation script relies on installation-specific UUIDs. To obtain the transfer source UUID for script invocation, visit the 'Transfer Source' tab in the Archivematica Storage Space web dashboard. If a row is marked as a transfer souce its UUID value will be valid for a transfer. To fully automate AIP storage transfer, the processingMCP.xml file used in your transfer must be changed. For example, if you are using the defaultProcessingMCP.xml file provided by this tool in the pre-transfer directory, you need to modify the following section:
+
+    <preconfiguredChoice>
+      <appliesTo>b320ce81-9982-408a-9502-097d0daa48fa</appliesTo>
+      <goToChain>/api/v2/location/99d0466e-173d-4b67-98a3-490198e56925/</goToChain>
+    </preconfiguredChoice>
```
Comment (Hwesta) [inline README.md]: As a note, another option is to generate a defaultProcessingMCP.xml in Archivematica itself, and copy that file to the automation tools.  This is probably how you'd change other choices (eg format identification, normalization etc) that are more difficult to modify when all you have to work with are UUIDs.

**Comment by Hwesta** [comment]
Thanks for the contribution, especially the documentation changes! Sorry it took me so long to get back to you. I requested a couple of tweaks - these can be squashed/amended into the current commits or as new commits.

**Comment by dnarc** [comment]
Moved my repo over to Bitbucket, I will re-submit this.

**Pull request closed without merging**